cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -239,6 +247,12 @@ func (manager *PipelineManager) Configure(ctx context.Context) error {
 		return err
 	}
 
+	// config pipeline handles setting or creating the provider pipeline to be used
+	err = manager.CiProvider.configurePipeline(ctx, gitRepoInfo, prj.Infra)
+	if err != nil {
+		return err
```

Comment:
Should this be `fmt.Errorf("configuring pipeline: %w")`?

Reply:
https://github.com/Azure/azure-dev/issues/746